__tests__/app.test.js

```diff
@@ -113,3 +113,40 @@ describe("GET api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("should respond with an array of comments with supplised review_id", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        expect(comments).toHaveLength(3);
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+        comments.forEach((comment) => {
+          expect.objectContaining({
+            comment_id: expect.any(Number),
+            body: expect.any(String),
+            author: expect.any(String),
+            review_id: expect.any(Number),
+            created_at: expect.any(Date),
+          });
```

Comment:
You should add a test for an existing article which has no comments. We would expect a 200 status and to return an empty array